### PR TITLE
REAC-326 Documentation updates

### PR DESCRIPTION
**Pull request by tommymorgan**
1. Restore a couple of files that were inadvertently overwritten with the wrong content
2. Remove a test spec file that was inadvertently added to the api-docs directory

### CODE REVIEW

File: api-docs/modern/modern/src/field/ComboBox.js [hunk 1]
```diff
@@ -1,255 +1,275 @@
...
- *          }]
- *      });
- *
- * See also the {@link #cfg!floatedPicker} and {@link #cfg!edgePicker} options for additional
- * configuration of the options list.
- *
- * @since 6.5.0
+ *     @example packages=[reactor]
+ *     import React, { Component } from 'react';
+ *     import { ExtReact, FormPanel, ComboBoxField, Container } from '@extjs/ext-react';
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/ComboBox.js]: Replace @extjs with @sencha

File: api-docs/modern/modern/src/field/Select.js [hunk 2]
```diff
@@ -1,362 +1,164 @@
...
- *         }]
- *     });
- */
-
-/**
- * @property {Boolean} isSelectField
- * `true` in this class to identify an object this type, or subclass thereof.
+ *     @example packages=[reactor]
+ *     import React, { Component } from 'react';
+ *     import { ExtReact, FormPanel, SelectField, Container } from '@extjs/ext-react';
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/Select.js]: replace @extjs with @sencha

File: api-docs/modern/modern/src/field/ComboBox.js [hunk 3]
```diff
@@ -1,255 +1,275 @@
...
- *   `{@link Ext.data.Store#cfg!remoteFilter remoteFilter}: true`, and further filters
- *   may be _programmatically_ added to the Store which are then passed with every
- *   load request which allows the server to further refine the returned dataset.
+ *   - **`'local'`** :
  *
- *   Typically, in an autocomplete situation, {@link #cfg!hideTrigger} is configured
- *   `true` because it has no meaning for autocomplete.
+ *     ComboBox loads local data
+ *
+ *         var combo = new Ext.form.field.ComboBox({
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/ComboBox.js]: Would you like to add an @example here?

**Comment by dalley** [comment]
@mgusmano can you review today?

File: api-docs/modern/modern/src/field/ComboBox.js (hunk 3 above)
Comment (tommymorgan) [reply api-docs/modern/modern/src/field/ComboBox.js]: I added an example.

File: api-docs/modern/modern/src/field/ComboBox.js (hunk 1 above)
Comment (tommymorgan) [reply api-docs/modern/modern/src/field/ComboBox.js]: Fixed

File: api-docs/modern/modern/src/field/Select.js (hunk 2 above)
Comment (tommymorgan) [reply api-docs/modern/modern/src/field/Select.js]: Fixed

**Comment by dalley** [comment]
@branflake2267 comments have been applied, please review.
@mgusmano Please review.

**Pull request merged**